worldedit_commands/cuboid.lua

```diff
@@ -202,13 +204,13 @@ worldedit.register_command("contract", {
 		worldedit.cuboid_linear_expand(name, axis, dir, -amount)
 		worldedit.cuboid_linear_expand(name, axis, -dir, -rev_amount)
 		worldedit.marker_update(name)
-		return true, "Region contracted by " .. (amount + rev_amount) .. " nodes"
+		return true, S("Region contracted by @1 nodes", amount + rev_amount)
 	end,
 })
 
 worldedit.register_command("cubeapply", {
 	params = "<size>/(<sizex> <sizey> <sizez>) <command> [parameters]",
-	description = "Select a cube with side length <size> around position 1 and run <command> on region",
+	description = S("Select a cube with side length <size> around position 1 and run <command> on region"),
 	privs = {worldedit=true},
 	require_pos = 1,
 	parse = function(param)
```

Comment:
below here you missed the "invalid usage:" string

Reply:
thnx, fixed